config/config.yaml

```diff
@@ -17,8 +17,8 @@ extensions:
     #    username: ${env:GRAF_USER}
     #    password: ${env:GRAF_PAT}
     ## Uncomment below if you want to enable authentication to GitLab
-    #bearertokenauth/gitlab:
-    #  token: ${env:GL_PAT}
+    bearertokenauth/gitlab:
```

Comment:
This should be commented out by default.

Reply:
If that is commented out, how is the PAT being used for gitlab? The graphQL client needed bearer auth token to work, at least on my end. Are we using another method of auth for the GitLab api?

Reply:
by default, you don't need to specify auth to make a query to the API. but you are more limited, and won't be able to see things that aren't public. You can see the screenshot of the postman request i made. it doesn't have any authentication specified but correctly returns information about my public group & project.

additionally, there are other fixes to this file. even if this was left uncommented, it wouldn't be used because of it not be specified in other parts of the file. the github stuff got removed & gitlab duplicated twice. we can pair on this or i can just push the fix. 

<img width="1299" alt="image" src="https://github.com/liatrio/liatrio-otel-collector/assets/25961386/79c81790-bc5a-46c2-820c-3eda80c8c67a">

Reply:
Yea, Sam and I made use of postman pretty heavily, but when we initially worked on this, we ran into issues with the genq client needing a token/pat of some sort to work which is why it was there to begin with. I guess I was just confused on the comment saying it was meant to be commented out, because I had to uncomment it for the genq client to work correctly. Unless there is another way that I am missing, but either way, we can pair on it or feel free to push the change.

Reply:
I'll do some testing to confirm it will work without a token and report back.

Reply:
the screenshot above is showing authless query of gitlab's graphql api endpoint. genqlient doesn't know about the requirement or auth itself. it just makes the same requests as code with an established client.